packages/framework/src/resources/workflow.ts

```diff
@@ -46,9 +46,10 @@ export function workflow<
       throw new MissingSecretKeyError();
     }
 
+    const unvalidatedData = (event.payload || {}) as T_PayloadUnvalidated;
```

Comment:
`payload` falls back to an empty object when it's not specified in args during `workflow(...).trigger(<args>)`